lib/Widget/evaluationForms/previsitForm/healthStatus.dart

```diff
@@ -0,0 +1,2715 @@
+                                      ),
+                                      Row(
+                                        children: [
+                                          Expanded(
+                                            child: Padding(
+                                              padding: EdgeInsets.all(
+                                                  screenSize.height / 70),
+                                              child: TextFormField(
+                                                keyboardType:
+                                                    TextInputType.multiline,
+                                                maxLines: null,
+                                                validator: (value) {
+                                                  return value.isEmpty
+                                                      ? 'กรุณากรอก Details for Abnormal'
+                                                      : null;
+                                                },
+                                                decoration: InputDecoration(
+                                                    contentPadding:
+                                                        new EdgeInsets
+                                                                .symmetric(
+                                                            vertical: 8.0,
+                                                            horizontal: 10.0),
+                                                    enabledBorder:
+                                                        OutlineInputBorder(
+                                                      borderSide: BorderSide(
+                                                          color: Colors.black26,
+                                                          width: 1),
+                                                    ),
+                                                    labelText:
+                                                        ' Details for Abnormal'),
+                                                onSaved: (value) =>
+                                                    _heent = value,
+                                              ),
+                                            ),
+                                          ),
+                                        ],
+                                      ),
+                                      Row(
+                                        children: [
+                                          Expanded(
+                                            flex: 2,
+                                            child: Padding(
+                                              padding: EdgeInsets.all(
+                                                  screenSize.height / 70),
+                                              child: Container(
+                                                child: Text('R.N. Signature:'),
```

Comment:
Please help to get the medical team name account to show as signature here

Reply:
@prawproudd  Done